examples/distributed-key-value-store.rs

```diff
@@ -188,7 +199,7 @@ fn handle_input_line(kademlia: &mut Kademlia<MemoryStore>, line: String) {
                 publisher: None,
                 expires: None,
             };
-            kademlia.put_record(record, Quorum::One);
+            kademlia.put_record(record, Quorum::One).expect("Failed to store record locally.");
```

Comment:
Instead of leaving the interpretation of the error up to the user (in this case the `expect` string, what do you think of having `RecordStore::put` return a proper error? Not saying this should happen within this pull request. I am happy to do that as a follow up.

Reply:
The interpretation isn't up to the user, the error type is a `store::Error`.